src/App.js

```diff
@@ -1,8 +1,27 @@
 import React from 'react'
-import EncounterPage from './EncounterPage/EncounterPage'
+import { BrowserRouter as Router, Switch, Route } from 'react-router-dom'
+import EncounterPage from './components/EncounterPage/EncounterPage'
+import CreatePage from './components/CreatePage/CreatePage'
+import useCreatureEntries from './components/hooks/useCreatureEntries'
 
 function App() {
-  return <EncounterPage />
+  const { creatureEntries, addCreatureEntry } = useCreatureEntries()
+  return (
+    <Router>
+      <Switch>
+        <Route path="/encounter">
+          <EncounterPage creatureEntries={creatureEntries} />
+        </Route>
+        <Route expect path="/">
+          <CreatePage
+            creatureEntries={creatureEntries}
+            addCreatureEntry={addCreatureEntry}
+          />
+          {/* <JournalPage entries={entries} addEntry={addEntry} /> */}
```

Comment:
if not needed, comment should be deleted.

Reply:
Oh, forgot to delete it, thanks ;)